database/migrations/2020_08_25_192340_create_offer_user_table.php

```diff
@@ -13,7 +13,7 @@ class CreateOfferUserTable extends Migration
      */
     public function up()
     {
-        Schema::create('offer_user', function (Blueprint $table) {
+        Schema::create('offer_users', function (Blueprint $table) {
```

Comment:
@RanaEmad  I think the common convention for naming the pivot table is using the singular name of each table. Please check the below links and let me know if I miss understood anything.
https://webdevetc.com/blog/laravel-naming-conventions
https://stackoverflow.com/questions/34897444/naming-tables-in-many-to-many-relationships-laravel/34897727

Reply:
The model was not updated with $table variable so it was breaking the code, I didn't know if it was on purpose or not so I adjusted it.

Reply:
@RanaEmad It is supposed to update the $table variable into the model, so please do that as I think the migrations already applied on the server so we cant update it using this way. If you need to update the table name please create a new migration file instead.